src/common/catalog/catalog-entity.ts

```diff
@@ -56,13 +56,14 @@ export abstract class CatalogCategory extends EventEmitter {
   };
   abstract spec: CatalogCategorySpec;
 
-  // Should be URL-compatible, otherwise "Uncaught TypeError: Expected "categoryId" to match "[^\/#\?]+?""
-  static getId(group: string, kind: string): string {
-    return `${group}--${kind}`;
+  static parseId(id = ""): { group?: string, kind?: string } {
+    const [group, kind] = id.split("/") ?? [];
+
+    return { group, kind };
   }
```

Comment:
This could be instead:

```ts
public getRouteParams(): ICatalogViewRouteParam {
  return {
	group: this.spec.group,
    kind: this.spec.names.kind,
  }
}
```

Reply:
It's `static` a method. It's done for that (to match with selected entity). See `catalog.tsx`

```tsx
  onTabChange = (categoryId: string) => {
    const { group, kind } = CatalogCategory.parseId(categoryId);

    if (group && kind) {
      navigate(catalogURL({ params: { group, kind } }));
    } else {
      navigate(catalogURL());
    }
  };
```

Reply:
It could also be:

```ts
  onTabChange = (categoryId: string) => {
    const activeCategory = this.categories.find(category => category.getId() === tabId);

    navigate(catalogURL({ params: activeCategory?.getRouteParams() }));
  };
```